service/src/main/java/uk/gov/hmcts/reform/fpl/model/DocmosisAnnexDocuments.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.model;
```

Comment:
Any reason for this not being in the docmosis package

Reply:
updated. good spot